src/pages/gen2/start/account-setup/index.mdx

```diff
@@ -9,105 +9,206 @@ export function getStaticProps(context) {
+9. Review the permission set and select **Create**
+10. After creation you are navigated back to the IAM Identity Center dashboard and are ready to create your first user. Using the navigation pane, select **Users**
+11. Enter the user details, then select **Next**
+
+  ![AWS IAM Identity Center user creation with the username "amplify-admin"](/images/gen2/account-setup/sso-create-user.png)
+
+12. Optionally create and add the user to a group, select **Next**
+13. Review the user information and select **Add user**. The user will then need to verify their email using the email specified during user creation.
+14. After user creation you are navigated back to the IAM Identity Center dashboard and are ready to assign the newly-created user to have access to an AWS account. For the sake of the demo we will use the AWS account we used to create the Organization, however you can create a new AWS account under your organization for use with Amplify. Check the management AWS account and select **Assign users or groups**
+
+  ![AWS IAM Identity Center "AWS accounts" page with the management account checked](/images/gen2/account-setup/sso-aws-accounts.png)
+
+15. When prompted to assign a user or group, select the **Users** tab, check the user created in step 13, and select **Next**
+  
+  ![AWS IAM Identity Center "AWS accounts" page assigning "amplify-admin" to the management AWS account](/images/gen2/account-setup/sso-aws-accounts.png)
+
+16. Assign the permission set created in step 9 and select **Next**
+17. Review the assignment information and select **Submit**
+18. Now you are ready to sign in to the access portal! Navigate back to the IAM Identity Center dashboard. Within the "Settings summary" pane, copy the URL for your "AWS access portal URL"
+
+  ![AWS IAM Identity Center dashboard highlighting the AWS access portal URL](/images/gen2/account-setup/sso-dashboard-access-portal.png)
+
+19. Navigate to the copied URL and sign in as your user, "amplify-admin". After signing in you should see you have access to an AWS account and are ready to get started!
 
-## Configure Amplify CI/CD
+  ![AWS IAM Identity Center access portal displaying an AWS account](/images/gen2/account-setup/sso-access-portal.png)
 
-Before you can start creating the cloud sandbox environment, Amplify will need to complete a one-time bootstrap setup for the account and region before it can start deploying resources in your AWS account.
+20. Back in your terminal, you are ready to configure an AWS profile that uses the SSO user. There are a few options for [getting IAM Identity Center user credentials](https://docs.aws.amazon.com/singlesignon/latest/userguide/howtogetcredentials.html), however we will use the AWS CLI configuration wizard. In your terminal, run `aws configure sso`
+
+<Callout info>
+
+[Learn how to install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
```

Comment:
```suggestion
Note this command is using the AWS CLI, not the Amplify CLI. [Learn how to install the AWS CLI here].(https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
```

Reply:
slightly tweaked this to bold "Note", but added